public/javascripts/application.js

```diff
@@ -25,6 +25,9 @@ $j(document).ready(function() {
     // make Approve buttons on inbox items visible
     $j('#inbox-form, .messages').find('.unreviewed').find('.review').find('a').removeClass('hidden');
 
+    // convert an absolute time to a relative time, e.g. 1 day ago
+    $j('.timeago').timeago();
```

Comment:
This fails if we're not on the admin user page.

Reply:
I think I have made the change you asked for.